### DISCUSSION TITLE
https://greendolphindance.com/2023/12/15/2023%E5%B0%8F%E7%BB%93/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#12] [General]
# https://greendolphindance.com/2023/12/15/2023%E5%B0%8F%E7%BB%93/

Back to the rough ground!

https://greendolphindance.com/2023/12/15/2023%E5%B0%8F%E7%BB%93/

**Comment by fujiwaranatsuho** [comment]
4m好像是在国金随便找的店子？因为一开始没想着喝咖啡

**Reply by greendolphindance** [reply]
好像是哦！想起来了

**Comment by Notizen23** [comment]
bug是胳膊手部的透视吗？

**Reply by greendolphindance** [reply]
不是！其实从侧面看手臂是好的
